app/assets/stylesheets/grey-branding.scss

```diff
@@ -108,4 +108,15 @@ thead th{
 /* Headers dropdown margin changes*/
 a#language-menu, a#reference-menu, a#user-menu{
   margin-bottom: 3px !important;
-}
\ No newline at end of file
+}
+
+/* Notification close icon visibility and position*/
+.close {
+  opacity: 1 !important;
+}
+
+button.close{
+  position: relative;
+  margin-top: -7px;
+  margin-right: -12px;
+}
```

Comment:
Opacity looks good, I just wonder about the margin-top value.
I tried `margin-top: 12px;` and am seeing this. What do you think?

![Screenshot from 2023-10-04 15-57-47](https://github.com/portagenetwork/roadmap/assets/71047780/0945c4f7-d9c0-40b9-89a5-5b3b9c3273bd)